charts/seed-controlplane/charts/etcd/templates/etcd-hvpa.yaml

```diff
@@ -34,18 +31,36 @@ spec:
         metrics:
         - resource:
             name: memory
-            targetAverageUtilization: {{ .Values.targetAverageUtilization }}
+            targetAverageUtilization: {{ .Values.targetAverageUtilizationMemory }}
           type: Resource
         - resource:
             name: cpu
-            targetAverageUtilization: {{ .Values.targetAverageUtilization }}
+            targetAverageUtilization: {{ .Values.targetAverageUtilizationCpu }}
           type: Resource
-    updatePolicy:
-      updateMode: "Auto"
   vpa:
     selector:
       matchLabels:
         role: etcd-vpa-{{ .Values.role }}
+    deploy: true
+    scaleUp:
+      updatePolicy:
+        updateMode: "Auto"
+{{- if .Values.scaleUpStabilization }}
+{{ toYaml .Values.scaleUpStabilization | indent 6 }}
+{{- end }}
+    scaleDown:
+      updatePolicy:
+        updateMode: "MaintenanceWindow"
+{{- if .Values.scaleDownStabilization }}
+{{ toYaml .Values.scaleDownStabilization | indent 6 }}
+{{- end }}
+    limitsRequestsGapScaleParams:
+      cpu:
+        value: "3"
```

Comment:
What do you think of making theses values configurable via chart values and putting the defaults in `values.yaml`?